electron_app/package.json

```diff
@@ -5,11 +5,17 @@
   "version": "1.4.2",
   "description": "A feature-rich client for Matrix.org",
   "author": "New Vector Ltd.",
+  "scripts": {
+    "build": "electron-build-env --electron 6.0.3 neon build seshat-node --release",
```

Comment:
The thing doesn't even work with the version passed as is. It seems to be only necessary if you need to rebuild seshat for some reason (e.g. if you're linking it from your dev folder) anyways.

Since we're adding run-time checks for seshat anyways it will be safe to remove it as well.